go/mysql/datetime/types.go

```diff
@@ -66,13 +67,32 @@ func (t Time) Format(prec uint8) []byte {
 }
 
 func (t Time) FormatInt64() (n int64) {
-	v := int64(t.Hour())*10000 + int64(t.Minute())*100 + int64(t.Second())
+	tr := t.Round(0)
```

Comment:
Can you please add a comment for this line? I can see why we'd want to round to the nearest second for Int32 but it's not obvious to me for Int64. Perhaps because MySQL unixtime values are 32 bit / second precision *when an integer type* (vs float which has microsecond precision).... (and this is, after all, the mysql/datetime package :-)). But then why not use FormatInt32()? Perhaps because we prefer to use int64's everywhere. We can clarify in a short comment.